scripts/rabbitmq-server-ha.ocf

```diff
@@ -1630,6 +1634,36 @@ get_monitor() {
     return $rc
 }
 
+ocf-update-attr() {
+    local attr_name="${1:?}"
+    local attr_value="{2:?}"
```

Comment:
Fixed.

On Thu, Jun 2, 2016 at 7:38 PM, Dmitry Mescheryakov <
notifications@github.com> wrote:

> In scripts/rabbitmq-server-ha.ocf
> <https://github.com/rabbitmq/rabbitmq-server/pull/819#discussion_r65574639>
> :
>
> > @@ -1630,6 +1634,36 @@ get_monitor() {
> >      return $rc
> >  }
> >
> > +ocf-update-attr() {
> > +    local attr_name="${1:?}"
> > +    local attr_value="{2:?}"
>
> Seems like you have forgotten '$' sign here
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rabbitmq/rabbitmq-server/pull/819/files/6eb69925f1a01368cc2e9d00a26c4d50b207921e#r65574639>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AALUY9sBAA7w-wZGj0RszoeXXszj79Uxks5qHwbzgaJpZM4IsvOD>
> .
>